src/tile-plotting.jl

```diff
@@ -126,7 +147,7 @@ function cull_plots!(m::Map)
     end
 end
 
-function create_tile_plot!(m::AbstractMap, tile::Tile, data)
+function create_tyler_plot!(m::AbstractMap, tile::Tile, data)
```

Comment:
Reminder to self: we need to document this name change and apply it to the GeoMakie extension + all other Tyler plot configs

Reply:
Oh is this interface?

Reply:
I checked, and the GeoMakie extension is using `create_tileplot` which is different from this `create_tile_plot` function.  Let's keep `create_tyler_plot`.